src/sentry/static/sentry/app/views/performance/traceDetails/content.tsx

```diff
@@ -266,6 +267,12 @@ class TraceDetailsContent extends React.Component<Props, State> {
   }
 
   renderTraceView(traceInfo: TraceInfo) {
+    const sentryTransaction = Sentry.getCurrentHub().getScope()?.getTransaction();
```

Comment:
This is instrumented on the entire trace view rather than the individual spans because
1. each individual span is fast enough that the span is essentially 0ms long
2. creating a span per transaction can easily exceed the span limit